### PR TITLE
add icons for light/dark toggle

### PR DESCRIPTION
**Pull request by cheapsteak**

### CODE REVIEW

File: src/icons/svgs/controls/icon-moon.svg [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<svg width="26px" height="26px" viewBox="0 0 26 26" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g transform="translate(-777.000000, -1903.000000)">
+            <g transform="translate(778.000000, 1904.000000)">
```
Comment (Jephuff) [inline src/icons/svgs/controls/icon-moon.svg:4]: Want to collapse these groups? :D 
should be something like
```suggestion
    <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" transform="translate(1 1)">
```

Comment (Jephuff) [inline src/icons/svgs/controls/icon-moon.svg:4]: Want to collapse these groups? :D 
should be something like
```suggestion
    <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" transform="translate(1, 1)">
```

File: src/icons/svgs/controls/icon-sun.svg [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<svg width="26px" height="26px" viewBox="0 0 26 26" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g class="icon-sun-group" transform="translate(-776.000000, -1852.000000)" stroke="#000000" stroke-width="1.5">
+            <g transform="translate(777.000000, 1853.000000)">
```
Comment (Jephuff) [inline src/icons/svgs/controls/icon-sun.svg:4]: Want to collapse these groups as well? :D 
```suggestion
    <g stroke="none" stroke-width="1.5" fill="none" fill-rule="evenodd" stroke="#000000" transform="translate(1, 1)">
```

File: src/icons/svgs/controls/icon-moon.svg (hunk 1 above)
Comment (cheapsteak) [reply src/icons/svgs/controls/icon-moon.svg:4]: it looks like we run these through svgo as part of the build process?

**Comment by cheapsteak** [comment]
@Jephuff ran the input through svgo

File: src/icons/svgs/controls/icon-moon.svg (hunk 1 above)
Comment (Jephuff) [reply src/icons/svgs/controls/icon-moon.svg]: ahh did not know that! :)

**Comment by apollo-bot2** [comment]
:rocket: PR was released in [`v6.1.0`](https://github.com/apollographql/space-kit/releases/tag/v6.1.0) :rocket:

**Pull request merged**